### DISCUSSION TITLE
TabularPredictor random seed

### DISCUSSION THREAD

**Discussion by smrutib2017** [autogluon/autogluon#4882] [Q&A]
I don't see an option to set a random_seed in TabularPredictor (unlike TimeSeriesPredictor). Is there a reason for that? If I want to control the random seed that's used by the underlying models in TabularPredictor, is there a way I can do that?

Thanks

**Comment by chaofan520** [comment]
Same question.

**Comment by aabragan** [comment]
I use sklearn `train_test_split` to do the random seed
![image](https://github.com/user-attachments/assets/9babc891-044d-4313-be8e-834e5e2669fb)
